Add documentation for ICorProfilerInfo13 methods and enumeration. (#30131)
dotnet/docs: docs/framework/unmanaged-api/profiling/icorprofilerinfo13-destroyhandle-method.md

---
description: "Learn more about: ICorProfilerInfo13::DestroyHandle Method"
title: "ICorProfilerInfo13::DestroyHandle Method"
ms.date: "03/19/2021"
api_name: 
  - "ICorProfilerInfo13.DestroyHandle"
api_location: 
  - "coreclr.dll"
  - "corprof.idl"
api_type: 
  - "COM"
---
# ICorProfilerInfo13::DestroyHandle Method

> Destroy a handle wrapping an object.

Reviewer: ```suggestion
Destroys a handle that wraps an object.
```

  
## Syntax  
  
```cpp  
    HRESULT DestroyHandle([in] ObjectHandleID handle);
```  
  
## Parameters

`handle`
[in] The handle returned by `CreateHandle`.

## Remarks

> Once destroyed, a handle cannot be used anymore.

Reviewer: ```suggestion
Once destroyed, a handle can no longer be used.
```
